e2e-tests/playwright/e2e/default-global-header.spec.ts

```diff
@@ -0,0 +1,115 @@
+import { expect, test } from "@playwright/test";
+import { UIhelper } from "../utils/ui-helper";
+import { Common } from "../utils/common";
+
+test.describe("Default Global Header", () => {
+  let common: Common;
+  let uiHelper: UIhelper;
+
+  test.beforeEach(async ({ page }) => {
+    uiHelper = new UIhelper(page);
+    common = new Common(page);
+    await common.loginAsKeycloakUser(
+      process.env.GH_USER2_ID,
+      process.env.GH_USER2_PASS,
+    );
+    await expect(page.locator("nav[id='global-header']")).toBeVisible();
+  });
+
+  test("Verify that global header and default header components are visible", async () => {
+    expect(await uiHelper.isSearchBarVisible()).toBeTruthy();
+    expect(await uiHelper.isLinkVisibleByLabel("Create...")).toBeTruthy();
+    expect(
+      await uiHelper.isLinkVisibleByLabel("Support (external link)"),
+    ).toBeTruthy();
+    expect(await uiHelper.isLinkVisibleByLabel("Notifications")).toBeTruthy();
+    expect(await uiHelper.isBtnVisible("rhdh-qe-2")).toBeTruthy();
+  });
+
+  test("Verify that search modal and settings button in sidebar are not visible", async () => {
+    expect(await uiHelper.isBtnVisible("Search")).toBeFalsy();
+    expect(await uiHelper.isBtnVisible("Settings")).toBeFalsy();
+  });
+
+  test("Verify that clicking on Create button opens the Software Templates page", async () => {
+    await uiHelper.clickLinkByAriaLabel("Create...");
+    await uiHelper.verifyHeading("Software Templates");
+  });
+
+  test("Verify that clicking on Support button opens a new tab", async ({
+    context,
+  }) => {
+    const [newTab] = await Promise.all([
+      context.waitForEvent("page"),
+      uiHelper.clickLinkByAriaLabel("Support (external link)"),
+    ]);
+    expect(newTab).not.toBeNull();
+    await newTab.waitForLoadState();
+    expect(newTab.url()).toContain(
+      "https://github.com/redhat-developer/rhdh/issues",
+    );
+    await newTab.close();
+  });
+
+  test("Verify Profile Dropdown behaves as expected", async ({ page }) => {
+    const profileDropdown = page.locator(
+      "[data-testid='KeyboardArrowDownOutlinedIcon']",
+    );
+    await profileDropdown.click();
+    expect(await uiHelper.isLinkVisible("Settings")).toBeTruthy();
```

Comment:
```suggestion
   await uiHelper.verifyLink("Settings");
```
similarly can we replace all expect(await isLinkVisible.()) as above?